policy/datasets.yaml

```diff
@@ -458,6 +458,17 @@ isr_metadata:
     python: lsst.daf.base.PropertySet
     tables: raw
     template: ''
+isrWrap_config:
```

Comment:
Please add a `description` docstring for these two, to distinguish them from the `isr_*` items just above.

Reply:
Description added.